<!DOCTYPE HTML>
<html lang="en">
  <head>
    <meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="Description" content="The Rafael Casellas Lab combines molecular biology, gene targeting, and transgenic techniques, as well as genomic and bioinformatic tools to unravel molecular mechanisms driving early development and peripheral activation of B lymphocytes"> 
    <meta name="keywords" content="Rafael,Casellas,Regulome,genomics,B cells,NIAMS">

    <title>NIH Regulome Timeline</title>

        <link rel="stylesheet" href="/assets/css/bootstrap.css" />
        <link rel="stylesheet" href="/assets/css/justified-nav.css" />
        <link rel="stylesheet" href="/assets/css/font-awesome.css" />
        
	<style>
	.container {
		font-family:"Palatino Linotype";
		font-size:18px;
	}
}
	</style> 
<!--[if lt IE 9]>
      <script src="/assets/js/html5shiv.js"></script>
      <script src="/assets/js/respond.js"></script>  
<![endif]--> 

  </head>
<!--  <body style="background-color: rgb(108,123,139);"> -->
	<body style="background-color: rgb(255,255,255);">

  <div class="container">
	<!-- Navigation Bar and header
	===================================================  -->
      <div class="masthead" style="padding-bottom:0px;">
        <ul class="nav nav-justified">
          <li><a href="https://www.nibib.nih.gov/research-funding/regulome-project">Home</a></li>
          <li class="active"><a href="/Timeline/index.html" data-toggle="tooltip">Timeline</a></li>
          <li><a href="/Publications/index.html">Publications</a></li>
          <li><a href="/People/index.html">Team</a></li>
	  <li style="white-space: nowrap"><a href="/Protocol/index.html">Protocols</a></li> 
          <li><a href="/Art/index.html">Art</a></li>
          <li><a href="/Join/index.html">Join</a></li>
        </ul>
      </div> <!-- /masthead -->
<!-- climate change  code start -->
<!--
<div id="rel_div" style="width:200px; z-index:100px; position:fixed; top:100px; right:50px; float:right; overflow: auto; ">
            <a href="https://climate.nasa.gov/", target="_blank"><img src="/images/bear.jpg" width="200"></a>
</div>
-->
<!-- climate change  code end -->

	<!-- Slideshow
	===================================================  -->
	<div class="container" style="background-color:white;font-family:'Helvetica';font-size:22px;padding-right:20px;padding-left:20px;"> 
<br>

<!--
<h3 style="font-family:'Helvetica';text-align:center;pading-left:20px;padding-right:20px;color: rgb(0,94,128);"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24360274" style="font-size:36px;">Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation</a></h3>

<div class="row" style="padding-left:40px;padding-right:20px;">
	<div class="col-md-12 text-center">
		<img src="/images/RegulomeCover.jpg" style="height:650px;">
	</div>
</div>


<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">A key finding of the ENCODE project is that the enhancer landscape of mammalian cells undergoes marked alterations during ontogeny. However, the nature and extent of these changes are unclear. As part of the NIH Mouse Regulome Project, we here combined DNaseI hypersensitivity, ChIP-seq, and ChIA-PET technologies to map the promoter-enhancer interactomes of pluripotent ES cells and differentiated B lymphocytes. We confirm that enhancer usage varies widely across tissues. Unexpectedly, we find that this feature extends to broadly transcribed genes, including Myc and Pim1 cell-cycle regulators, which associate with an entirely different set of enhancers in ES and B cells. By means of high-resolution CpG methylomes, genome editing, and digital footprinting, we show that these enhancers recruit lineage-determining factors. Furthermore, we demonstrate that the turning on and off of enhancers during development correlates with promoter activity. We propose that organisms rely on a dynamic enhancer landscape to control basic cellular functions in a tissue-specific manner.</h3>

<br>

<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Raw datasets the ChIP-seq data in activated B cells (other than CTCF) and CH12 cells, DHS-seq in activated B cells, RNA-seq, whole genome methylation (other than ES cells), and ChIA-PET reported in this paper can be accessed through the Short Read Archive with the accession number <a href="http://www.ncbi.nlm.nih.gov/sra/?term=SRP029721">SRP029721</a>.</h3>

<br>

<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Tracks for ChIA-PET, DHS hotspots, DHS footprints, and CpG methylation for activated B cells and ES cells are available <a href="http://figshare.com/articles/Tracks/893601">here</a>.(Due to the large file size, you may have to reload the download multiple times).
<br>Please note that the CHIA-PET files include all individual PETs, which at some loci amount to more than a hundred. For reasons of space PETs were clustered in the manuscript figures as explained in Figure S2A and the Methods section.</h3>

<br>
<p>
<h3 style="font-family:'Helvetica';padding-right:50px;padding-left:50px;font-size:28px;"><img src="/test/images/logo.jpg" alt="regulome logo" width="430" height="410" style="float:left;"><br><br> Following Francis Collins emphasis on intramural-extramural collaborations and the application of cutting-edge tools to research, the NIH Mouse Regulome Project was established in 2010. The primary objective of this program is to elucidate how gene expression is regulated in the mouse genome by combining the latest technology developed at intramural and extramural NIH labs. At each stage of the project different extramural investigators are invited to participate.<br><br></h3>
</p>

<br><br>
-->
<h3 style="color:blue;font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Phase I: 2010-2013</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Rafael Casellas (NIAMS-NCI), Keji Zhao (NHLBI), Lino Tesarrollo (NCI), David Levens (NCI), Teresa Przyticka (NLM), and David Schatz (Yale).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Main findings:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Using ES and B cells we uncovered a new phenomenon we dubbed transcriptome amplification, whereby the entire transcriptional program of G0 cells is augmented ~10-fold as they enter the cell cycle. We showed that the oncoprotein Myc and the TFIIH complex play a key role in this process.<br><br>
Through ChIP-seq analysis of the V(D)J recombinase complex RAG1-2, we defined recombination centers where antibody and TCR genes are assembled in developing lymphocytes. We showed that RAG2 binds to thousands of sites in the genome, including those involved in chromosomal translocations, thus providing a rationale to the role of RAGs in lymphomagenesis.</p>
</h3>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20398922" style="color:black"><i>Ji et al.</i> The in vivo pattern of binding of RAG1 and RAG2 to antigen receptor loci. <b>Cell</b>, 2010.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/23021216" style="color:black"><i>Nie et al.</i> c-Myc is a universal Amplifier of Expressed Genes in Lymphocytes and Embryonic Stem Cells. <b>Cell</b>, 2012.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24706737" style="color:black"><i>Kouzine et al.</i> Global Regulation of Promoter Melting in B lymphocytes. <b>Cell</b>, 2013.</a>
</h3>
<br>

<h3 style="color:blue;font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Phase II: 2013-2014</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Rafael Casellas (NIAMS-NCI), Gordon Hager (NCI), Yijun Ruan (Jackson labs), Keith Joung (Harvard), Michel Nussenzweig (Rockefeller), Qiang Pan-Hammarstrom (Karolinska Institute), and Fred Alt (Harvard).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Main findings:</u></h3>
<h3 style="font-family:'Helvetica';margin-top:1.5em;margin-bottom:1.5em;padding-right:20px;padding-left:20px;">Using ChIA-PET technology we revealed that as cells differentiate during embryonic development, promoters for some broadly expressed genes (e.g. the oncoproteins Pim1, Myc) become tethered to a completely different set of enhancers, demonstrating that the DNA regulatory landscape of mammalian cells is highly dynamic. <br><br>
We showed that the B cell deaminase AID, responsible for recombination and hypermutation of immunoglobulin genes, is promiscuously recruited by highly interactive super-enhancer domains, a feature that helps explain why B lymphocytes in mice and humans are particularly prone to chromosomal translocations and tumor development.
</h3>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24360274" style="color:black"><i>Kieffer-Kwon et al.</i> Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation. <b>Cell</b>, 2013.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/25483776" style="color:black"><i>Meng et al.</i> Convergent Transcription at Intragenic Super-Enhancers Targets AID-Initiated Genomic Instability.  <b>Cell</b>, 2014.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/25483777" style="color:black"><i>Qian et al.</i> B Cell Super-Enhancers and Regulatory Clusters Recruit Tumorigenic Activity. <b>Cell</b>, 2014</a>
</h3>
<br>

<h3 style="color:blue;font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Phase III: 2014-2017</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Rafael Casellas (NIAMS-NCI), Andre Nussenzweig (NCI), Gordon Hager (NCI), Hari Shroff (NIBIB), Erez Lieberman Aiden (Baylor), David Schartz (Yale), Yijun Ruan (Jackson labs), Lino Tessarrollo (NCI), Monique Floer (Michigan), Melike Lakadamyali (Barcelona), and Zhe Liu (Janelia Farm, HHMI).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Main findings:</u></h3>
<h3 style="font-family:'Helvetica';margin-top:1.5em;margin-bottom:1.5em;padding-right:20px;padding-left:20px;">We found that the genome of na√Øve B lymphocytes is poised. Namely, that RNA PolII, TFs, chromatin remodeling complexes, histone acetyltransferases, methyltransferases, and nuclear architectural proteins, are preloaded in G0 chromatin. Their activity however is maintained at basal levels until their catalytic substrates or cofactors, including the Myc oncoprotein, reach an optimal concentration upon antigen encounter. <br><br>
We showed that RAG proteins, which are highly promiscuous in their binding, can induce DNA breaks at active regulatory elements harboring recombination signal sequences in developing B and T cells, thus supporting the notion that these enzymes are involved in the etiology of chromosomal translocations. <br><br>
Using a new cohesin-degron system, we showed that all loop domains are eliminated upon acute depletion of cohesin rings. Furthermore, contrary to expectation, loss of architectural loops deregulates expression of a fraction of genes. <br><br>
We showed that cohesin extrusion renders the mammalian genome vulnerable to topoisomerase-induced DNA breaks. Importantly, we demonstrated that these lesions become substrates for chromosomal translocations in various human tumors. <br><br>
We defined a new architectural feature we dubbed "stripes", whereby cohesin extrusion tethers super-enhancers to cognate promoters and facilitates transcription of oncogenes and recombination of immunoglobulin genes.
</h3>
<br>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/26234156" style="color:black"><i>Teng et al.</i> RAG represents a widespread threat to the lymphocyte genome <b>Cell</b>, 2015.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/28803781" style="color:black"><i>Kieffer-Kwon et al.</i> Myc regulates chromatin decompaction and nuclear architecture during B cells activation. <b>M. Cell</b>, 2017.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/28985562" style="color:black"><i>Rao et al.</i> Cohesin loss eliminates all loop domains. <b>Cell</b>, 2017.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/28735753" style="color:black"><i>Canela et al.</i> Genome organization drives chromosome fragility. <b>Cell</b>, 2017</a><br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/29706548" style="color:black"><i>Vian et al.</i> The energetics and physiological impact of cohesin extrusion. <b>Cell</b>, 2018</a>
</h3>
<br>

<h3 style="color:blue;font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Phase IV: 2018-present</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Rafael Casellas (NIAMS-NCI), Fred Alt (Harvard), Gordon Hager (NCI), Hari Shroff (NIBIB), Dan Larson (NCI), Carson Chow (NIDDK), Aleksandra Pekowska (Nencki Institute), Erez Lieberman Aiden (Baylor), Yijun Ruan (Jackson labs), and Francisco Asturias (UC Denver).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Main findings and ongoing research:</u></h3>
<h3 style="font-family:'Helvetica';margin-top:1.5em;margin-bottom:1.5em;padding-right:20px;padding-left:20px;">We aim to understand how promoters and enhancers are tethered during gene expression in primary and tumor cells. The current model posits that regulatory elements form stable loops created by cohesin and specific transcription factors (e.g. YY1, Pax5) or transcriptional complexes (PolII-Mediator). Our studies however showed that Mediator and PolII do not contribute to regulatory DNA proximity. Ongoing efforts try to identify the contributing factors and measure their exact role in nuclear architecture.<br><br>
Regulatory elements in mammalian genomes activate promoters by recruiting transcription factors (TFs) to DNA motifs. Although motif arrangement is critical to their function, the syntax behind the cis-regulatory code in normal and transformed cells remains undefined. We are exploring this question in ~500 mouse and human cell types by combining an atlas of TF motifs with transcriptome, accessibility, and footprinting maps.<br><br>
Through live cell imaging, Hi-C, and mathematical modeling, we have defined the dynamics of TFF1 transcriptional bursting upon estrogen exposure. By incorporating single cell RNA-Seq, we are now expanding those studies to the entire mouse and human transcriptomes.<br><br>
We used cryo-EM to obtain a 5.5A map of mammalian Mediator. By increasing the resolution of this map to the atomic level, we continue our molecular studies of transcription initiation.  <br><br>
We have discovered that V(D)J and class switch recombination of antibody genes is driven by a process consistent with cohesin extrusion. We are currently using degron systems to validate such model in normal recombination and the formation of chromosomal translocations.</h3>
<br>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/31402173" style="color:black"><i>El Khattabi et al.</i> A pliable Mediator acts as a functional rather than an architecutral bridge between promoters and enhancers. <b>Cell</b>, 2019.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/30554876" style="color:black"><i>Rodriguez et al.</i> Intrinsic dynamics of a human gene reveal the basis of expression heterogeneity. <b>Cell</b>, 2019.</a> <br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/31511698" style="color:black"><i>Zhang et al.</i> The fundamental roles of chromatin loop extrusion in physiological V(D)J recombination. <b>Nature</b>, 2019</a><br><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/31666703" style="color:black"><i>Zhang et al.</i> Fundamental roles of chromatin loop extrusion in antibody class switching. <b>Nature</b>, 2019</a>
</h3>
<br>

<br>

</div> <!-- top container -->  

</div> <!-- end container -->
<br><br>        
</div>
<!--
<div class="container" style="margin: 0;padding-bottom:15px;padding-top:5px;background-color:white;font-family: 'Adobe Garamond Pro';">
<h5 style="text-align:right;"> Disclaimer: This is not an official web site of the National Institutes of Health.</h5>
</div>
-->
<!-- Google Analytics -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-47028203-1', 'casellaslab.github.io');
  ga('send', 'pageview');

</script>

                <!--[if lt IE 9]>
                        <script src="/assets/js/require.js"></script>
                <![endif]-->
        <script src="/assets/js/jquery.js"></script>
        <script src="/assets/js/bootstrap.js"></script>
        <!--[if lt IE 9]>
                     <script src="/assets/js/css3-mediaqueries.js"></script>
                <![endif]-->
  </body>
</html>
