<!DOCTYPE HTML>
<html lang="en">
  <head>
    <meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="Description" content="The Rafael Casellas Lab combines molecular biology, gene targeting, and transgenic techniques, as well as genomic and bioinformatic tools to unravel molecular mechanisms driving early development and peripheral activation of B lymphocytes"> 
    <meta name="keywords" content="Rafael,Casellas,Regulome,genomics,B cells,NIAMS">

    <title>Casellas Lab Site - NIAMS - Regulome</title>

        <link rel="stylesheet" href="/assets/css/bootstrap.css" />
        <link rel="stylesheet" href="/assets/css/justified-nav.css" />
        <link rel="stylesheet" href="/assets/css/font-awesome.css" />
        
	<style>
	.container {
		font-family:"Palatino Linotype";
		font-size:18px;
	}
}
	</style> 
<!--[if lt IE 9]>
      <script src="/assets/js/html5shiv.js"></script>
      <script src="/assets/js/respond.js"></script>  
<![endif]--> 

  </head>
<!--  <body style="background-color: rgb(108,123,139);"> -->
	<body style="background-color: rgb(255,255,255);">

  <div class="container">
	<!-- Navigation Bar and header
	===================================================  -->
      <div class="masthead" style="padding-bottom:0px;">
        <ul class="nav nav-justified">
          <li><a href="/index.html">Home</a></li>
          <li><a href="/Research/index.html" data-toggle="tooltip">Research</a></li>
          <li><a href="/People/index.html">People</a></li>
          <li><a href="/Publications/index.html">Publications</a></li>
          <li class="active"><a href="/Regulome/index.html">Regulome</a></li>
	  <li style="white-space: nowrap"><a href="/GenomeEditing/index.html">Genome Editing</a></li> 
          <li><a href="/Join/index.html">Join</a></li>
        </ul>
      </div> <!-- /masthead -->
<!-- climate change  code start -->
<!--
<div id="rel_div" style="width:200px; z-index:100px; position:fixed; top:100px; right:50px; float:right; overflow: auto; ">
            <a href="https://climate.nasa.gov/", target="_blank"><img src="/images/bear.jpg" width="200"></a>
</div>
-->
<!-- climate change  code end -->

	<!-- Slideshow
	===================================================  -->
	<div class="container" style="background-color:white;font-family:'Helvetica';font-size:22px;padding-right:20px;padding-left:20px;"> 
<br>

<!--
<h3 style="font-family:'Helvetica';text-align:center;pading-left:20px;padding-right:20px;color: rgb(0,94,128);"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24360274" style="font-size:36px;">Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation</a></h3>

<div class="row" style="padding-left:40px;padding-right:20px;">
	<div class="col-md-12 text-center">
		<img src="/images/RegulomeCover.jpg" style="height:650px;">
	</div>
</div>


<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">A key finding of the ENCODE project is that the enhancer landscape of mammalian cells undergoes marked alterations during ontogeny. However, the nature and extent of these changes are unclear. As part of the NIH Mouse Regulome Project, we here combined DNaseI hypersensitivity, ChIP-seq, and ChIA-PET technologies to map the promoter-enhancer interactomes of pluripotent ES cells and differentiated B lymphocytes. We confirm that enhancer usage varies widely across tissues. Unexpectedly, we find that this feature extends to broadly transcribed genes, including Myc and Pim1 cell-cycle regulators, which associate with an entirely different set of enhancers in ES and B cells. By means of high-resolution CpG methylomes, genome editing, and digital footprinting, we show that these enhancers recruit lineage-determining factors. Furthermore, we demonstrate that the turning on and off of enhancers during development correlates with promoter activity. We propose that organisms rely on a dynamic enhancer landscape to control basic cellular functions in a tissue-specific manner.</h3>

<br>

<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Raw datasets the ChIP-seq data in activated B cells (other than CTCF) and CH12 cells, DHS-seq in activated B cells, RNA-seq, whole genome methylation (other than ES cells), and ChIA-PET reported in this paper can be accessed through the Short Read Archive with the accession number <a href="http://www.ncbi.nlm.nih.gov/sra/?term=SRP029721">SRP029721</a>.</h3>

<br>

<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Tracks for ChIA-PET, DHS hotspots, DHS footprints, and CpG methylation for activated B cells and ES cells are available <a href="http://figshare.com/articles/Tracks/893601">here</a>.(Due to the large file size, you may have to reload the download multiple times).
<br>Please note that the CHIA-PET files include all individual PETs, which at some loci amount to more than a hundred. For reasons of space PETs were clustered in the manuscript figures as explained in Figure S2A and the Methods section.</h3>

<br>
-->
<p>
<h3 style="font-family:'Helvetica';padding-right:50px;padding-left:50px;font-size:28px;"><img src="/images/logo.jpg" alt="regulome logo" width="430" height="410" style="float:left;"><br><br> Following Francis Collins emphasis on intramural-extramural collaborations and the application of cutting-edge tools to research, the NIH Mouse Regulome Project was established in 2011. The primary objective of this program is to elucidate how gene expression is regulated in the mouse genome by combining the latest technology developed at intramural and extramural NIH labs. At each stage of the project different extramural investigators are invited to participate.<br><br></h3>
</p>

<br><br>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Stage I: 2011-2013</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Rafael Casellas (NIAMS-NCI), David Levens (NCI), Keji Zhao (NHLBI), Lino Tesarrollo (NCI), Teresa Przyticka (NLM), Jan Hoeijmakers (Erasmus Medical Center), and Wim Vermeulen (Erasmus Medical Center, the Netherlands).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Objectives and main findings:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Genomic tools were applied to explore how mRNA output is controlled during ontogeny. Using primary ES cells and B lymphocytes the studies revealed that transcriptomes are amplified as G0 cells enter the cell cycle. Two factors were identified as key drivers of transcriptome amplification: Myc and the TFIIH complex. In both cases the factors are maintained at basal levels in G0 but their expression is rapidly enhanced as cells are stimulated. Conversely, PolII is recruited at similar levels to G0 and cycling cell promoters.</h3>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/23706737" style="color:black"><i>Kouzine et al.</i> Global Regulation of Promoter Melting in B lymphocytes. <b>Cell</b>, 2013.</a>
</h3>
<br>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Stage II: 2013-2014</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Rafael Casellas (NIAMS-NCI), Yijun Ruan (Jackson labs), Gordon Hager (NCI), Keith Joung (Harvard), Michel Nussenzweig (Rockefeller), and Qiang Pan-Hammarstrom (Karolinska Institute).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Objectives and main findings:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Nuclear interconnectivity between gene regulatory elements was explored by means of genome editing, Digital Genomic Footprinting, and ChIA-PET technologies. A key finding was the observation that the enhancer landscape changes drastically during mammalian development. Even for broadly expressed genes (e.g. Pim1, Myc), promoters often associate with a completely different set of enhancers in progenitors (ES) and fully differentiated cells (B lymphocytes). A side finding was that highly active super-enhancers in B cells are highly susceptible to DNA damage, a feature that helps explain why B cells are particularly prone to the formation of chromosomal translocations and tumor development during the immune response.</h3>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24360274" style="color:black"><i>Kieffer-Kwon et al.</i> Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation. <b>Cell</b>, 2013.</a> <br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/25483777" style="color:black"><i>Qian et al.</i> B Cell Super-Enhancers and Regulatory Clusters Recruit Tumorigenic Activity. <b>Cell</b>, 2014</a>
</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Available Data:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Raw datasets the ChIP-seq data in activated B cells (other than CTCF) and CH12 cells, DHS-seq in activated B cells, RNA-seq, whole genome methylation (other than ES cells), and ChIA-PET reported in this paper can be accessed through the Short Read Archive with the accession number <a href="http://www.ncbi.nlm.nih.gov/sra/?term=SRP029721">SRP029721</a>.</h3>

<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Tracks for ChIA-PET, DHS hotspots, DHS footprints, and CpG methylation for activated B cells and ES cells are available <a href="http://figshare.com/articles/Tracks/893601">here</a>.(Due to the large file size, you may have to reload the download multiple times).
<br>Please note that the CHIA-PET files include all individual PETs, which at some loci amount to more than a hundred. For reasons of space PETs were clustered in the manuscript figures as explained in Figure S2A and the Methods section.</h3>
<br>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Stage III: 2014-present</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Rafael Casellas (NIAMS-NCI), Erez Lieberman Aiden (Baylor), Yijun Ruan (Jackson labs), Gordon Hager (NCI), Lino Tessarrollo (NCI), Monique Floer (Michigan), Melike Lakadamyali (Barcelona), Hari Schroff (NIBIB), Zhe Liu (Janelia Farm, HHMI), Fred Alt (Harvard), and Francisco Asturias (Scripps).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Objectives and main findings:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">The architecture of chromatin in B and ES cells is being explored by a combination of in situ HiC, ChIA-PET, genome editing, nanoscopy, single molecule microscopy, and EM technologies. Based on these approaches three manuscripts are currently being prepared which define the roles of histone acetylation and the Mediator complex in chromatin decondensation and nuclear topology. The ultimate goal is to assess at a single cell level how these processes control transcription in the mouse genome.</h3>
<br>

</div> <!-- top container -->  

</div> <!-- end container -->
<br><br>        
</div>
<!-- Google Analytics -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-47028203-1', 'casellaslab.github.io');
  ga('send', 'pageview');

</script>

                <!--[if lt IE 9]>
                        <script src="/assets/js/require.js"></script>
                <![endif]-->
        <script src="/assets/js/jquery.js"></script>
        <script src="/assets/js/bootstrap.js"></script>
        <!--[if lt IE 9]>
                     <script src="/assets/js/css3-mediaqueries.js"></script>
                <![endif]-->
  </body>
</html>
