<!DOCTYPE HTML>
<html lang="en">
  <head>
    <meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="Description" content="The Rafael Casellas Lab combines molecular biology, gene targeting, and transgenic techniques, as well as genomic and bioinformatic tools to unravel molecular mechanisms driving early development and peripheral activation of B lymphocytes"> 
    <meta name="keywords" content="Rafael,Casellas,Regulome,genomics,B cells,NIAMS">

    <title>Casellas Lab Site - NIAMS - Regulome</title>

        <link rel="stylesheet" href="/assets/css/bootstrap.css" />
        <link rel="stylesheet" href="/assets/css/justified-nav.css" />
        <link rel="stylesheet" href="/assets/css/font-awesome.css" />
        
	<style>
	.container {
		font-family:"Palatino Linotype";
		font-size:18px;
	}
}
	</style> 
<!--[if lt IE 9]>
      <script src="/assets/js/html5shiv.js"></script>
      <script src="/assets/js/respond.js"></script>  
<![endif]--> 

  </head>
<!--  <body style="background-color: rgb(108,123,139);"> -->
	<body style="background-color: rgb(255,255,255);">

  <div class="container">
	<!-- Navigation Bar and header
	===================================================  -->
      <div class="masthead" style="padding-bottom:0px;">
        <ul class="nav nav-justified">
          <li><a href="/test/index.html">Home</a></li>
          <li><a href="/test/Research/index.html" data-toggle="tooltip">Research</a></li>
          <li><a href="/test/People/index.html">People</a></li>
          <li><a href="/test/Publications/index.html">Publications</a></li>
          <li class="active"><a href="/test/Regulome/index.html">Regulome</a></li>
	  <li style="white-space: nowrap"><a href="/test/GenomeEditing/index.html">Protocols</a></li> 
          <li><a href="/test/Join/index.html">Join</a></li>
        </ul>
      </div> <!-- /masthead -->
<!-- climate change  code start -->
<!--
<div id="rel_div" style="width:200px; z-index:100px; position:fixed; top:100px; right:50px; float:right; overflow: auto; ">
            <a href="https://climate.nasa.gov/", target="_blank"><img src="/images/bear.jpg" width="200"></a>
</div>
-->
<!-- climate change  code end -->

	<!-- Slideshow
	===================================================  -->
	<div class="container" style="background-color:white;font-family:'Helvetica';font-size:22px;padding-right:20px;padding-left:20px;"> 
<br>

<!--
<h3 style="font-family:'Helvetica';text-align:center;pading-left:20px;padding-right:20px;color: rgb(0,94,128);"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24360274" style="font-size:36px;">Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation</a></h3>

<div class="row" style="padding-left:40px;padding-right:20px;">
	<div class="col-md-12 text-center">
		<img src="/images/RegulomeCover.jpg" style="height:650px;">
	</div>
</div>


<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">A key finding of the ENCODE project is that the enhancer landscape of mammalian cells undergoes marked alterations during ontogeny. However, the nature and extent of these changes are unclear. As part of the NIH Mouse Regulome Project, we here combined DNaseI hypersensitivity, ChIP-seq, and ChIA-PET technologies to map the promoter-enhancer interactomes of pluripotent ES cells and differentiated B lymphocytes. We confirm that enhancer usage varies widely across tissues. Unexpectedly, we find that this feature extends to broadly transcribed genes, including Myc and Pim1 cell-cycle regulators, which associate with an entirely different set of enhancers in ES and B cells. By means of high-resolution CpG methylomes, genome editing, and digital footprinting, we show that these enhancers recruit lineage-determining factors. Furthermore, we demonstrate that the turning on and off of enhancers during development correlates with promoter activity. We propose that organisms rely on a dynamic enhancer landscape to control basic cellular functions in a tissue-specific manner.</h3>

<br>

<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Raw datasets the ChIP-seq data in activated B cells (other than CTCF) and CH12 cells, DHS-seq in activated B cells, RNA-seq, whole genome methylation (other than ES cells), and ChIA-PET reported in this paper can be accessed through the Short Read Archive with the accession number <a href="http://www.ncbi.nlm.nih.gov/sra/?term=SRP029721">SRP029721</a>.</h3>

<br>

<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Tracks for ChIA-PET, DHS hotspots, DHS footprints, and CpG methylation for activated B cells and ES cells are available <a href="http://figshare.com/articles/Tracks/893601">here</a>.(Due to the large file size, you may have to reload the download multiple times).
<br>Please note that the CHIA-PET files include all individual PETs, which at some loci amount to more than a hundred. For reasons of space PETs were clustered in the manuscript figures as explained in Figure S2A and the Methods section.</h3>

<br>
-->
<p>
<h3 style="font-family:'Helvetica';padding-right:50px;padding-left:50px;font-size:28px;"><img src="/test/images/logo.jpg" alt="regulome logo" width="430" height="410" style="float:left;"><br><br> Following Francis Collins emphasis on intramural-extramural collaborations and the application of cutting-edge tools to research, the NIH Mouse Regulome Project was established in 2010. The primary objective of this program is to elucidate how gene expression is regulated in the mouse genome by combining the latest technology developed at intramural and extramural NIH labs. At each stage of the project different extramural investigators are invited to participate.<br><br></h3>
</p>

<br><br>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Stage I: 2010-2013</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">David Levens (NCI), Keji Zhao (NHLBI), Lino Tesarrollo (NCI), Teresa Przyticka (NLM), Jan Hoeijmakers and Wim Vermeulen (Erasmus Medical Center, the Netherlands).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Objectives and main findings:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Using ES and B cells we uncovered a new phenomenon we called transcriptome amplification, whereby the entire transcriptional program of G0 cells is augmented ~10-fold as they enter the cell cycle. Myc and the TFIIH complex play a key role in this process.</h3>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/23021216" style="color:black"><i>Nie et al.</i> c-Myc is a universal Amplifier of Expressed Genes in Lymphocytes and Embryonic Stem Cells. <b>Cell</b>, 2012.</a> <br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24706737" style="color:black"><i>Kouzine et al.</i> Global Regulation of Promoter Melting in B lymphocytes. <b>Cell</b>, 2013.</a>
</h3>
<br>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Stage II: 2013-2014</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Yijun Ruan (Jackson labs), Gordon Hager (NCI), Keith Joung (Harvard), Michel Nussenzweig (Rockefeller), Qiang Pan-Hammarstrom (Karolinska Institute), and Fred Alt (Harvard).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Objectives and main findings:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Using 3C technology we showed that as cells differentiate during embryonic development, promoters for some broadly expressed genes (e.g. Pim1, Myc) become tethered to a completely different set of enhancers, demonstrating that the DNA regulatory landscape of mammalian cells is highly dynamic. <br>
Another finding was that the B cell deaminase AID, responsible for recombination and hypermutation of Ig genes, is promiscuously recruited by highly interactive super-enhancer domains, a feature that helps explain why lymphocytes are particularly prone to chromosomal translocations and tumor development.
</h3>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24360274" style="color:black"><i>Kieffer-Kwon et al.</i> Interactome maps of mouse gene regulatory domains reveal basic principles of transcriptional regulation. <b>Cell</b>, 2013.</a> <br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/25483776" style="color:black"><i>Meng et al.</i> Convergent Transcription at Intragenic Super-Enhancers Targets AID-Initiated Genomic Instability.  <b>Cell</b>, 2014.</a> <br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/25483777" style="color:black"><i>Qian et al.</i> B Cell Super-Enhancers and Regulatory Clusters Recruit Tumorigenic Activity. <b>Cell</b>, 2014</a>
</h3>
<br>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Stage III: 2014-2017</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Erez Lieberman Aiden (Baylor), Andre Nussenzweig (NCI), Yijun Ruan (Jackson labs), Gordon Hager (NCI), Lino Tessarrollo (NCI), Monique Floer (Michigan), Melike Lakadamyali (Barcelona), Hari Schroff (NIBIB), Zhe Liu (Janelia Farm, HHMI).</h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Objectives and main findings:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">These studies revealed that the genome of na√Øve B lymphocytes is poised. Namely, that RNA PolII, TFs, chromatin remodeling complexes, histone acetyltransferases, methyltransferases, and nuclear architectural proteins, are preloaded in G0 chromatin. Their activity however is maintained at basal levels until their catalytic substrates or cofactors reach an optimal concentration upon antigen encounter. <br>
On another set of studies we showed that the activities behind nuclear architecture in B and other cell types render the genome vulnerable to topoisomerase-induced DNA breaks. We also found evidence that these lesions become substrates for chromosomal translocations in human cancer. <br>
Our studies have also defined a new architectural feature we dubbed "stripes" which primarily tether super-enhancers to cognate promoters and facilitates Ig gene transcription and recombination.
</h3>
<br>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>References:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">
<a href="http://www.ncbi.nlm.nih.gov/pubmed/28803781" style="color:black"><i>Kieffer-Kwon et al.</i> Myc regulates chromatin decompaction and nuclear architecture during B cells activation. <b>M. Cell</b>, 2017.</a> <br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/28735753" style="color:black"><i>Canela et al.</i> Genome organization drives chromosome fragility. <b>Cell</b>, 2017</a><br>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/29706548" style="color:black"><i>Vian et al.</i> The energetics and physiological impact of cohesin extrusion. <b>Cell</b>, 2018</a>
</h3>
<br>

<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:32px;"><b>Stage IV: 2018-present</b></h3>
<h3 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Participating Laboratories:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;">Erez Lieberman Aiden (Baylor), Yijun Ruan (Jackson labs), Francisco Asturias (UC Denver), Fred Alt (Harvard), Anshul Kundaje (Stanford), and Felipe Cortes Ledesma (Seville).</h3>
<!--<h4 style="font-family:'Helvetica';padding-left:20px;padding-right:20px;font-size:28px;"><u>Objectives and main findings:</u></h3>
<h3 style="font-family:'Helvetica';padding-right:20px;padding-left:20px;"> 
<ul>
<li>Generating a TF binding map of the mouse and human genomes by means of ATAC- and DHS-Seq, neural networks and CRISPR techniques.</li>
<li>Studying the role of Mediator in PolII activation by cryo-EM and genetic analysis.</li>
<li>Exploring the role of epigenetic accessibility in the tethering of regulatory DNA by means of in-situ HiC.</li>
</ul>
</h3>
-->
<br>

</div> <!-- top container -->  

</div> <!-- end container -->
<br><br>        
</div>
<!-- Google Analytics -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-47028203-1', 'casellaslab.github.io');
  ga('send', 'pageview');

</script>

                <!--[if lt IE 9]>
                        <script src="/assets/js/require.js"></script>
                <![endif]-->
        <script src="/assets/js/jquery.js"></script>
        <script src="/assets/js/bootstrap.js"></script>
        <!--[if lt IE 9]>
                     <script src="/assets/js/css3-mediaqueries.js"></script>
                <![endif]-->
  </body>
</html>
